lib/full-site-editing/edit-site-page.php

```diff
@@ -139,6 +139,12 @@ function gutenberg_edit_site_init( $hook ) {
 		wp_enqueue_style( 'wp-block-library-theme' );
 	}
 
+	// Remove wp-reset-editor-styles, as it's not needed in the iframed site editor.
+	if ( isset( $wp_styles->registered['wp-edit-blocks'] ) ) {
+		$wp_edit_blocks_dependencies                   = array_diff( $wp_styles->registered['wp-edit-blocks']->deps, array( 'wp-reset-editor-styles' ) );
```

Comment:
I don't fully understand why `wp-reset-editor-styles` has to be removed as a dependency to `wp-edit-blocks`. Given that, what you're doing here seems like an intuitive way to fix the problem.

I am curious if @ellatrix or @youknowriad have further thoughts on this, especially since they wrote the original patches to avoid injecting 'wp-reset-editor-styles' into the iframe.

Reply:
See #33204.